frontend/packages/ceph-storage-plugin/src/plugin.ts

```diff
@@ -210,6 +210,23 @@ const plugin: Plugin<ConsumedExtensions> = [
       },
     },
   },
+  // Download External Cluster Metadata
+  {
+    type: 'ClusterServiceVersion/Action',
+    properties: {
+      kind: 'StorageCluster',
+      label: 'Download Cluster Metadata',
+      apiGroup: models.OCSServiceModel.apiGroup,
+      callback: (_kind, _obj) => () =>
+        import(
+          './components/converged-credentials/credentials' /* webpackChunkName: "ceph-storage-export-credentials" */
+        )
+          .then((m) => m.default({}))
+          .catch((e) => {
+            throw e;
```

Comment:
Is this really needed?

webpack should still throw an "uncaught error from promise" if I'm not mistaken.

(Also applies to existing use of `ClusterServiceVersion/Action` extension in code above.)

Reply:
kind, obj are in the type signature of callback in console-sdk.
I think we don't need to throw the error.